pyblish_lite/control.py

```diff
@@ -95,6 +95,10 @@ def on_next():
 
         util.defer(100, on_next)
 
+    def controlEmit(self, signal, kwargs):
```

Comment:
The argument signature resembles [that of pyblish-qml](https://github.com/pyblish/pyblish-qml/blob/635c82d75fa5d5e294339bf8265f05eb5ca3b5d3/pyblish_qml/rpc/service.py#L121) and that's good. I'd prefer `**kwargs` over `kwargs`, but the reason it isn't is because the RPC interface (xmlrpc) doesn't support variable length argument signatures.